README.md

```diff
@@ -11,15 +11,19 @@ Similar to `Object.assign`, but with React static keywords blacklisted from
 being overridden.
 
 ```bash
-$ npm install --save hoist-non-react-statics
+# install package
+npm install --save hoist-non-react-statics
+
+# install typings
+typings install --save --global npm:hoist-non-react-statics/index.d.ts
 ```
 
 ## Usage
 
-```js
-import hoistNonReactStatic from 'hoist-non-react-statics';
+```ts
+import hoistNonReactStatic = require('hoist-non-react-statics')
```

Comment:
This is typescript rather than ES6/CommonJS. And the compile `module` option is `commonjs`.

The reason is that this module export with:

```js
module.exports = function() { /* ... */ }
```

This is same to

```ts
export = function() { /* ... */ }
```

in typescript.

It is not ES6 syntax, is ES6, we can export with

```js
export var a = 1
// or export default:
var b = 2
export default b
```

And then will be transform into CommonJS as follow:

```js
export.a = 1

var b = 2
export.default = b
```

See <https://www.typescriptlang.org/docs/handbook/modules.html>